README.md

```diff
@@ -34,12 +34,18 @@
 
 4. You have 2 options to install the project
 #### docker-compose based installation
-This approach will use only docker-compose to install Storefront project. Files between the host and guest are synced with "delegated" options (see FILE_SYNC in .env) 
+If you want to run search-service please first - populate `auth.json` in the root of repository with your magento composer keys
```

Comment:
auth.json file is missing in the root of repo

Reply:
removed